### DISCUSSION TITLE
How do you improve librdKafka producer performance?

### DISCUSSION THREAD

**Discussion by VisonZh** [edenhill/librdkafka#3422] [Q&A]
If there are 10,000,000 messages that need to be continuously produced and consumed at the same time, how to configure producer configuration attributes and consumer configuration attributes？

**Comment by edenhill** [comment]
https://github.com/edenhill/librdkafka/blob/master/INTRODUCTION.md#performance

**Reply by VisonZh** [reply]
thanks！I have another question. Can Kafka be used as a middleware for a real-time messaging system? Can the delay be guaranteed within 200ms ?

**Comment by VisonZh** [comment]
@edenhill please help！！
